cmd/crictl/config.go

```diff
@@ -39,9 +39,10 @@ EXAMPLE:
 CRICTL OPTIONS:
 	 runtime-endpoint:	Container runtime endpoint
 	 image-endpoint:	Image endpoint
-	 timeout:	Timeout of connecting to server
-	 debug:	Enable debug output
-	 pull-image-on-create: 	Enable pulling image on create requests`,
+	 timeout:	Timeout of connecting to server (default: 2s)
+	 debug:	Enable debug output (default: false)
+	 pull-image-on-create:	Enable pulling image on create requests (default: false)
+	 disable-pull-on-run:	Disable pulling image on run requests (default: false)`,
```

Comment:
Do we wanna inverse it and make `pull-on-run` where the default is `true`?

Reply:
thought about it.. went with the bools all have a default of false coin flip

Reply:
it seemed odd to have some bools be false when unset and this one true when unset...

Reply:
perhaps if we had a config default command to generate / show the defaults at the cli

Reply:
Hm yeah I think the current implementation is fine then. :)